### DISCUSSION TITLE
Bad Request Error when Adding new users and devices

### DISCUSSION THREAD

**Discussion by SnowTail101** [netbox-community/netbox#14177] [Help Wanted!]
Hi

I recently installed Netbox on a local server and am trying to get started with adding items. I noticed that I get a Bad Request Error when adding new devices or users  etc in Netbox online interface. I am signed in with the superuser account. My colleague who also logs in with the same superuser account on his machine doesn't have this issue. Are there any restrictions on the superuser account login? Does one machine take priority over another for instance? Any advice to get around this issue will me much appreciated. 

Netbox Version is v3.6.2
I am using both IE and Chrome browsers. same issue when using a private window and after clearing cookies and cache from the browsers.
<img width="240" alt="Capture" src="https://github.com/netbox-community/netbox/assets/147928844/40bca7aa-de57-46e9-8f77-d28d27437386">

**Comment by candlerb** [comment]
There are no restrictions on superuser.

You have a problem, but you've haven't shown any details of the problem (e.g. what error you see). Nor have you said what version of Netbox you're using.

Without that information, it's impossible to guess what's wrong with your system.

**Reply by SnowTail101** [reply]
I have now updated the question. Thanks for pointing out the missing info. First post asking for help

**Comment by candlerb** [comment]
Do you have any logs from Netbox or from Nginx? 400 is just a generic "the client sent an invalid request" error.

A 400 error can commonly be returned if:
1. You haven't set `ALLOWED_HOSTS` in configuration.py to match the hostname you're connecting to (compare this setting with what's in your browser's URL bar)
2. You haven't configured Nginx properly to set the `X-Forwarded-Host:` header. You can find a valid sample nginx.conf [here](https://github.com/netbox-community/netbox/blob/develop/contrib/nginx.conf). 

However, since the 400 error page appears to be generated by Nginx itself, it might be something else.

**Reply by SnowTail101** [reply]
thank you for your response, very good suggestions;

1.  ALLOWED_HOSTS setting didn't seem to make a difference, I tried both asterisk '*' and matching the url
2.  nginx.conf settings all look good

The fact that another colleague can  login with the superuser account and doesn't see this problem, feels like the setup is all working ok in general and this is a local machine issue. To check, I tried to login with at different user credentials, the issue pursues.  I am the main administrator and need to start creating tokens etc. at the moment it is not possible. The only other thing I can try is testing it on other machines in the office..

**Comment by mtinberg** [comment]
I know this sounds basic, but have you tried resetting your browser cache, or using an incognito window to test.  I've had some very​ strange behavior from webapps on occasion due to incorrectly cached assets or cookies.


—

Mark Tinberg ***@***.***>

Division of Information Technology-Network Services

University of Wisconsin-Madison


________________________________

**Reply by SnowTail101** [reply]
Hi  Mark. Yes, I know what you mean with cache and cookies. Unfortunately, I have tried those already and didn't fix the problem. I was convinced that would have been the issue but no luck.

**Comment by SnowTail101** [comment]
I have resolved the issue; it was due to the web browsers. Both Edge and Chrome is still giving the error despite clearing cache and cookies multiple times. I have installed a third browser and it is running fine on Firefox browser. Thank you everyone for your suggestions, I will close this now.
